core/domain/stats_jobs_one_off.py

```diff
@@ -217,6 +221,19 @@ def reduce(exp_id, stringified_values):
             elif value['event_type'] == feconf.EVENT_TYPE_STATE_HIT:
                 state_name = value['state_name']
                 session_id = value['session_id']
+
+                # Explicit logging of state names with + characters insted of
```

Comment:
Some state names in events have spaces replaced with plus signs. We explicitly log these for future reference.

Reply:
Done.